### PR TITLE
doc: Fix typos in dfa.rs

### PR DESCRIPTION
**Pull request by koba-e964**
Minor typos are fixed.

### CODE REVIEW

File: src/dfa.rs [hunk 1]
```diff
@@ -913,8 +913,8 @@ impl<'a> Fsm<'a> {
         if self.state(si).flags().has_empty() {
             // Compute the flags immediately preceding the current byte.
             // This means we only care about the "end" or "end line" flags.
-            // (The "start" flags are computed immediately proceding the
-            // current byte and is handled below.)
+            // (The "start" flags are computed immediately preceding the
```
Comment (BurntSushi) [inline src/dfa.rs:916]: This should be "following."

File: src/dfa.rs [hunk 2]
```diff
@@ -1048,7 +1048,7 @@ impl<'a> Fsm<'a> {
     ///
     /// If matching starts after the beginning of the input, then only start
     /// line should be set if the preceding byte is `\n`. End line should never
-    /// be set in this case. (Even if the proceding byte is a `\n`, it will
+    /// be set in this case. (Even if the preceding byte is a `\n`, it will
```
Comment (BurntSushi) [inline src/dfa.rs:1051]: This should also be "following."

**Review by BurntSushi — CHANGES_REQUESTED** [review]
Thanks! Instances of "proceding" should be "following" though.

**Comment by koba-e964** [comment]
I think I fixed the issue. https://github.com/rust-lang/regex/pull/717/commits/890ece7e89365235361a297dbc99cc0ca2e3a74f

**Review by BurntSushi — APPROVED** [review]
Great, thanks!

**Pull request merged**